src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(male =>
+    male.sex === 'm'
+    && (century > 0 ? Math.ceil(male.died / 100) === century : true)
+  );
+
+  return men
+    .reduce((count, male) => count += (male.died - male.born), 0)
```

Comment:
The tests fail because in this arrow function `count += (male.died - male.born)` basically says `return count += (male.died - male.born)`. You can see the problem now. You are returning an assignment. Remove the `=`.